src/app/pages/ParaTimesPage/TransactionAmount/index.tsx

```diff
@@ -91,6 +92,7 @@ export const TransactionAmount = () => {
       <Box margin={{ bottom: 'medium' }}>
         <Form
           messages={{ required: t('paraTimes.validation.required', 'Field is required') }}
+          noValidate
```

Comment:
how come?

Reply:
I don't want to show a default html validation triggered by min and step HTML attrs